plugins/catalog-backend/migrations/20201007201501_index_entity_search.js

```diff
@@ -21,8 +21,17 @@
  */
 exports.up = async function up(knex) {
   await knex.schema.alterTable('entities_search', table => {
-    table.index(['key'], 'entities_search_key');
-    table.index(['value'], 'entities_search_value');
+    if (knex.client.config.client.includes('mysql')) {
+      table.index(['key'], 'entities_search_key', {
+        indexType: 'FULLTEXT',
+      });
+      table.index(['value'], 'entities_search_value', {
+        indexType: 'FULLTEXT',
+      });
+    } else {
```

Comment:
could you replace this with something like

```ts
const options =
  knex.client.config.client.includes('mysql')
    ? { indexType: 'FULLTEXT' }
    : {};
```

and then pass that into the two original `table.index`?

Reply:
Will do!